### PR TITLE
Kelsonic 33027 tweak header v2 auth

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
This PR implements some tweaks to the design when authenticated for header v2.

## Original issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/33027

## Testing done
N/A

## Screenshots
Screen Shot 2021-11-17 at 1 09 14 PM
Screen Shot 2021-11-17 at 1 09 27 PM

## Acceptance criteria
- [x] Update auth header v2 design

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/platform/site-wide/header/components/LogoRow/styles.scss [hunk 1]
```diff
@@ -12,6 +12,23 @@
   .profile-nav-contianer > div {
     padding-top: 0 !important;
   }
+
+  .sign-in-drop-down-panel-button {
+    border-top: none !important;
```
Comment (oseasmoran73) [inline src/platform/site-wide/header/components/LogoRow/styles.scss:17]: Is there a way we can apply without using `!important`? Applies for all in the PR

Comment (kelsonic) [reply src/platform/site-wide/header/components/LogoRow/styles.scss:17]: Nope, unfortunately

Comment (kelsonic) [reply src/platform/site-wide/header/components/LogoRow/styles.scss:17]: All of this code for styling the user nav will be moot once I get the time to rewrite it for v2

**Review by srsuddath — APPROVED** [review]

File: src/platform/site-wide/header/components/LogoRow/styles.scss (hunk 1 above)
Comment (oseasmoran73) [reply src/platform/site-wide/header/components/LogoRow/styles.scss:17]: :sad-parrot: Just wanted to double check!

**Review by oseasmoran73 — APPROVED** [review]

File: src/platform/site-wide/header/components/LogoRow/styles.scss (hunk 1 above)
Comment (kelsonic) [reply src/platform/site-wide/header/components/LogoRow/styles.scss:17]: totally! it was a great comment! 🔥 I am really looking forward to removing them, I am not a fan of `!important` 😂

**Pull request merged**